internal/controller/broker/broker_facade.go

```diff
@@ -24,48 +24,42 @@ const (
 	BrokerLabelValue = "true"
 )
 
-//go:generate mockery -name=brokerSyncer -output=automock -outpkg=automock -case=underscore
-type brokerSyncer interface {
-	SyncServiceBroker(namespace string) error
-}
-
-// BrokersFacade is responsible for creation k8s objects for namespaced broker
-type BrokersFacade struct {
-	client           client.Client
-	workingNamespace string
-	serviceName      string
-	log              logrus.FieldLogger
-
-	brokerSyncer brokerSyncer
+// Facade is responsible for creation k8s objects for namespaced broker
+type Facade struct {
+	client          client.Client
+	systemNamespace string
+	serviceName     string
+	namespace       string
+
+	log logrus.FieldLogger
 }
 
 // NewBrokersFacade returns facade
-func NewBrokersFacade(cli client.Client, brokerSyncer brokerSyncer, workingNamespace, serviceName string, log logrus.FieldLogger) *BrokersFacade {
-	return &BrokersFacade{
-		client:           cli,
-		workingNamespace: workingNamespace,
-		brokerSyncer:     brokerSyncer,
-		serviceName:      serviceName,
-		log:              log.WithField("service", "nsbroker-facade"),
+func NewBrokersFacade(cli client.Client, systemNamespace, serviceName string, log logrus.FieldLogger) *Facade {
```

Comment:
`NewBrokersFacade` returns new `Facade`, is it on purpose?

Reply:
Yes, using it from the external package it will look like that: `broker.Facade{}`

I changed it because of the `go vet` issue with naming